src/components/TrackTile/TrackerDetails/TrackerDetails.test.tsx

```diff
@@ -84,7 +92,8 @@ describe('Tracker Details', () => {
     });
   });
 
-  it.skip('should upsert the tracker with a new target on unmount', async () => {
+  it('should upsert the tracker with a new target on unmount', async () => {
```

Comment:
I had meant to circle back and re-enable this test and fix it but it got away from me. Thank you for doing this!